### PR TITLE
changed: rewrite of the plugin for Elgg 3.0

### PR DESCRIPTION
**Pull request by rohit1290**

### CODE REVIEW

File: CHANGES.txt [hunk 1]
```diff
@@ -1,6 +1,10 @@
 Version history
 ===============
 
+3.1
+
+- changes for Elgg 3.0
+
```
Comment (jeabakker) [inline CHANGES.txt]: Please don't touch changelog as this is done upon release by a script of ours

File: elgg-plugin.php [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<?php
+
+return [
+	'actions' => [
+		'account_removal/choices' => [],
+		'account_removal/confirm' => [],
+	],
+	'routes' => [
+		'collection:account_removal:confirm' => [
+			'path' => '/account_removal/confirm/{username}/{type}',
```
Comment (jeabakker) [inline elgg-plugin.php]: this is different from the current url structure. Please update the usage where url's are regenated (Page menu and email notification)

File: elgg-plugin.php [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+			'resource' => 'account_removal/confirm',
+		],
+		'collection:account_removal:choices' => [
+			'path' => '/account_removal/choices/{username}',
```
Comment (jeabakker) [inline elgg-plugin.php]: see comment above

File: manifest.xml [hunk 4]
```diff
@@ -2,7 +2,7 @@
 <plugin_manifest xmlns="http://www.elgg.org/plugin_manifest/1.8">
 	<name>Account Removal</name>
 	<author>ColdTrick IT Solutions</author>
-	<version>3.0</version>
+	<version>3.1</version>
```
Comment (jeabakker) [inline manifest.xml]: Please don't touch version number as this is handled upon release of the plugin

PS: maybe for dev purpose make it 4.0-beta

**Review by jeabakker — CHANGES_REQUESTED** [review]
Thanks for the effort, please look at the comments.

File: CHANGES.txt (hunk 1 above)
Comment (rohit1290) [reply CHANGES.txt]: reverted back to original file

File: elgg-plugin.php (hunk 2 above)
Comment (rohit1290) [reply elgg-plugin.php]: updated

File: elgg-plugin.php (hunk 3 above)
Comment (rohit1290) [reply elgg-plugin.php]: updated

File: manifest.xml (hunk 4 above)
Comment (rohit1290) [reply manifest.xml]: updated

**Comment by jeabakker** [comment]
thanks for the work, I'll probably do some code cleanup in a little while

**Pull request merged**